README.md

```diff
@@ -49,7 +58,74 @@ The `fairlearn` package contains the following algorithms for mitigating unfairn
+
+
+
+The following plots provide a view of the configuration steps, where `gender` is selected as a protected attribute and `accuracy` is selected as the metric of interest to be analyzed for different subgroups of `gender`.
+
+
+
+![Configuration Steps](img/fairlearn-dashboard-config.png)
+
+
+
+
+
+|Plot|Description|
+|----|-----------|
+|Disparity in Accuracy| This panel (1) shows the accuracy of your model with respect to your selected accuracy metric (e.g., `accuracy`) (A) on the entire model, and (B) on different subgroups of your selected protected attribute (e.g., `accuracy` for females, `accuracy` for males, `accuracy` for datapoints with a non-binary gender), (2) highlights the disparity (difference) of selected accuracy metric across different subgroups of protected attribute. It also (3) shows the distribution of errors in each subgroup (e.g., female, male, non-binary gender). In the binary classification case, it will further split the errors into overprediction (predicting 1 when the true label is 0), and underprediction (predicting 0 when the true label is 1).
+|Disparity in Predictions|This panel shows a bar chart that contains the selection rate in each group. This means the fraction of points classified as 1 (in binary classification) and distribution of prediction values in regression.|
+
+
+
+The following plots provide a view of the final evaluation step.
+
+
+![Fairness Insights](img/fairlearn-dashboard-results.png)
+
+### Visualization: multiple models, post-mitigation 
+
+Post-mitigation visualization dashboard has been mainly designed for the reduction mitigation techniques `GridSearch` and `ExponentiatedGradient` where mitigation phase produces an array of mitigated models. However, it can work with post-processing approach `ThresholdOptimizer` to showcase comparative chart of pre and post mitigation models.
+
+
+The post-mitigation visualization dashboard consists of the same procedural steps for configuration and evaluation of model fairness insights. You still go through the configuration stage, select your protected attribute and accuracy metric of interest. After you select your parameters, before landing on the evaluation page, you are shown a screen that compares optimal tradeoffs between computed models' error values and disparity metrics (accuracy vs. disparity) for your original unmitigated predictor and all computed mitigated predictors (output of the reduction technique or the models generated by the post-processing approach). You can then observe the trade-off (if exists) and compare the performance vs. fairness of your different models, and click on any of the predictors to land on the evaluation page for that selected predictor.
```

Comment:
'You still go' is another sudden change in voice/style.

Reply:
Agreed. I think I mostly fixed style in my pass.